pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -96,6 +98,15 @@ def get_highest_version(self, obj):
         )
         return {"href": href, "version": version}
 
+    def get_download_count(self, obj):
+        """Get the download count of the collection"""
+        dlc_model = models.CollectionDownloadCount
+        if dlc_model.objects.filter(namespace=obj.namespace, name=obj.name).count() == 0:
+            return 0
+        return (
+            dlc_model.objects.filter(namespace=obj.namespace, name=obj.name).first().download_count
+        )
```

Comment:
Lets use a reusable queryset...
```suggestion
        qs = models.CollectionDownloadCount.objects.filter(namespace=obj.namespace, name=obj.name)
        if qs.count() == 0:
            return 0
        return qs.first().download_count
```

Reply:
Wow, how'd I miss that.  Thanks!